### PR TITLE
docs(DEVELOPERS): add link to ergo repo - I659

### PR DESCRIPTION
**Pull request by Parikshit-Hooda**
Signed-off-by: Parikshit Hooda <phooda804@live.com>

# Issue #659 
link added to the main ergo github repo.

### Changes
- added a link to the main ergo github repo which initially seemed like a placeholder for the link.

### Flags
- <DESCRIBE ISSUES OR HOLDS FOR REVIEWERS>

### Related Issues
- Issue #659 
- Pull Request #<NUMBER HERE>

### CODE REVIEW

**Comment by mttrbrts** [comment]
@jeromesimeon @Parikshit-Hooda , the original values for the links are correct (they are defined at the bottom of the file). However, they should not be escaped.

Please simply replace `\[` and `\]` with `[` and `]`

File: DEVELOPERS.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ npm install -g lerna@2.11.0
 
 To contribute code to Ergo, you must have a GitHub account so you can push code to your own fork of Ergo and open Pull Requests in the \[GitHub Repository\]\[github\].
 
-To create a Github account, follow the instructions [here](https://github.com/signup/free). Afterwards, go ahead and [fork](http://help.github.com/forking) the \[main Ergo repository\]\[github\].
+To create a Github account, follow the instructions [here](https://github.com/signup/free). Afterwards, go ahead and [fork](http://help.github.com/forking) the [main Ergo repository](Github).
```
Comment (mttrbrts) [inline DEVELOPERS.md]: Thanks for reverting the change @Parikshit-Hooda, however the short-code needs to match the list at the bottom of the file exactly, i.e. `[main Ergo repository][github]`

**Comment by mttrbrts** [comment]
@Parikshit-Hooda, can you also fix the other instances of  `\[` and `\]` in this file, please?

Also, your last merge commit is missing the DCO sign-off, running `git commit --amend -s` and then `git push --force` should fix it for you.

**Comment by jeromesimeon** [comment]
@Parikshit-Hooda Any progress on this?

**Comment by Parikshit-Hooda** [comment]
hi @jeromesimeon On it. Shall push in a day or two.

**Pull request closed without merging**